src/main/kotlin/app/revanced/patches/tiktok/misc/settings/patch/TikTokSettingsPatch.kt

```diff
@@ -79,12 +83,39 @@ class TikTokSettingsPatch : BytecodePatch(
             )
             break
         }
+        //Change onClick of About to start revanced settings
+        val method2 = AboutOnClickMethodFingerprint.result!!.mutableMethod
+        method2.addInstructions(
+            0,
+            """
+                    invoke-static {}, Lapp/revanced/tiktok/settingsmenu/SettingsMenu;->startSettingsActivity()V
+                    return-void
+                """
+        )
+        //Replace string `About` to 'Revanced Settings` in TikTok settings.
+        for ((index, instruction) in implementation1.instructions.withIndex()) {
```

Comment:
Instead of searching for the instruction index, either use an opcode pattern or use the string scan result from the fingerprint and a fixed offset.

Reply:
You could try and match the first one and do some heuristic offsetting from there (try to find your instruction from the matching point, don't directly index unless you can guarantee it'll work across versions).

Reply:
@Sculas There are more bytecode which have same opcode pattern in target method. The pic only shows one of them.